scripts/deploy.mjs

```diff
@@ -0,0 +1,41 @@
+import { exec } from 'child_process';
+import * as ghPages from 'gh-pages';
+
+const branch = process.env.GITHUB_REF.replace('refs/heads/', '');
+const deployEnv = branch.match(
+  /^main|^release|^production|^hotfix|^lab-\d+/
+)?.[0];
```

Comment:
@kavimaluskam I found I don't like Prettier that much (or current setting of Prettier).
E.g. I found this kind of breaking lines are hard to read.

Reply:
Of course i am used to the breaking of prettier already... shall we research on the configuration of prettier later?

I think one advantage of prettier is enforcing us to use the same styling during development so commit / fix about styling is avoided.

Reply:
I agree there is advantage of this.
But this `?.[0]` is broken into another line, which I still strange.